docs/tutorials/error_analysis.rst

```diff
@@ -0,0 +1,1199 @@
+====================
+
+Set the pulsar system's coordinates
+:math:`(\alpha_\mathrm{p}, \delta_\mathrm{p})` and proper motion components
+:math:`(\mu_\mathrm{p,sys,\alpha\ast}, \mu_\mathrm{p,sys,\delta})`,
+as well as some of the system's parameters that are known from timing studies:
+its orbital period :math:`P_\mathrm{orb,p}`, projected semi-major axis
+:math:`a_\mathrm{p} \sin( i_\mathrm{p} )`, and radial-velocity amplitude
+:math:`K_\mathrm{p} = 2 \pi a_\mathrm{p} \sin( i_\mathrm{p} )
+/ P_\mathrm{orb,p}` [which relates to the pulsar's mean orbital speed as
+:math:`v_\mathrm{0,p} = K_\mathrm{p} / \sin( i_\mathrm{p} )`].
+
+.. jupyter-execute::
+
+    psr_coord = SkyCoord('04h37m15.99744s -47d15m09.7170s',
+                         pm_ra_cosdec=121.4385 * u.mas / u.yr,
+                         pm_dec=-71.4754 * u.mas / u.yr)
+    
+    p_orb_p = 5.7410459 * u.day
+    asini_p = 3.3667144 * const.c * u.s
+    
+    k_p = 2.*np.pi * asini_p / p_orb_p
+
+Set the known properties of Earth's orbit (the orbital period :math:`P_\oplus`,
+its semi-major axis :math:`a_\oplus`, and the mean orbital speed
+:math:`v_{0,\oplus} = 2 \pi a_\oplus / P_\mathrm{orb,\oplus}`), and derive its
+orientation with respect to the line of sight (i.e., the orbit's inclination
+:math:`i_\oplus` and longitude of ascending node :math:`\Omega_\oplus`).
+
+.. jupyter-execute::
+
+    p_orb_e = 1. * u.yr
+    a_e = 1. * u.au
+
+    v_0_e = 2.*np.pi * a_e / p_orb_e
+    
+    psr_coord_eclip = psr_coord.barycentricmeanecliptic
+    ascnod_eclip = SkyCoord(lon=psr_coord_eclip.lon - 90.*u.deg, lat=0.*u.deg,
+                            frame='barycentricmeanecliptic')
+    ascnod_equat = ascnod_eclip.icrs
+    
+    i_e = psr_coord_eclip.lat + 90.*u.deg
+    omega_e = psr_coord.position_angle(ascnod_equat)
+
+.. warning::
+
+    This calculation assumes that Earth's orbit is circular, which is of course
+    not completely accurate. As noted above, the pulsar's orbit is also assumed
+    to be circular. These simplifications result in a model in which it is
+    clear how the scintillation velocities depend on the physical parameters
+    of the system, but this model can clearly be improved by implementing more
+    realistic orbits for the pulsar and Earth.
+
+
+List the parameters and their properties
+========================================
+
+This tutorial deals with three different sets of parameters:
+
+- the harmonic coefficients used in the fitting :math:`(A_\mathrm{\oplus,s},
+  A_\mathrm{\oplus,c}, A_\mathrm{p,s}, A_\mathrm{p,c}, C)`,
+- the sinusoid amplitudes and phase offsets of the phenomenological model
+  :math:`(A_\oplus, A_\mathrm{p}, \chi_\oplus, \chi_\mathrm{p}, C)`,
+- the physical parameters :math:`(i_\mathrm{p}, \Omega_\mathrm{p},
+  d_\mathrm{p}, d_\mathrm{s}, \xi, v_\mathrm{lens,\parallel})`.
+
+Here, we simply define lists of strings used for printing results and
+labelling plots later.
+
+.. jupyter-execute::
+
+    var_strs_harc = [
```

Comment:
I'd define the `var_strs` as a list of 2-element tuples, with name and unit as elements. Could even use a `named_tuple`...